README.md

```diff
@@ -33,10 +33,11 @@ For Ethereum network data you can either run a local node or use Infura.io:
 
 ### Running a local Graph Node
 
-This is a quick example to get you up and running, it uses a [subgraph for ENS](https://github.com/graphprotocol/ens-subgraph) that we built as a reference.
+This is a quick example to to show a working graph node. It is a [subgraph for the Ethereum name Service (ENS)](https://github.com/graphprotocol/ens-subgraph) that the graph team built.
```

Comment:
Capitalize `graph` (2x)
`Ethereum name Service` -> `Ethereum Name Service`

Reply:
Actually, capitalize both words in `Graph Node`, like we do in the rest of the README.